test/unit/file-utils.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
```

Comment:
If we're going to have different utils. Then this can be in a utils folder and simplify the name `/test/unit/utils/file.js`